cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -147,14 +185,14 @@ function UnmanagedClusterNetworkSettings(props: Partial<StepProps>) {
                 </div>
                 <div cds-layout="col:4">
                     <CdsInput layout="vertical" control-width="shrink">
-                        <label cds-layout="p-b:md">
+                        <label>
                             Cluster pod CIDR <CdsIcon shape="info-circle" size="md" status="info"></CdsIcon>
                         </label>
                         <input
                             {...register(UNMANAGED_NETWORK_FIELDS.CLUSTER_POD_CIDR)}
-                            placeholder="Cluster Pod CIDR"
+                            placeholder="Cluster POD CIDR"
                             onChange={handleFieldChange}
-                            defaultValue="100.96.0.0/11"
+                            defaultValue={'100.96.0.0/11'}
```

Comment:
Suggest defining a constant for the hard-coded string

Reply:
This was supposed to use the values from the store I mustve changed it by accident